Many editorial fixes (rollup 1) (#867)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

defined in {{I-D.ietf-taps-arch}}. The interface defined in this document
provides:

- Access to a variety of transport protocols, independent
  of the Protocol Stacks that will be used at
  runtime. All common features of these protocol
  stacks are made available to the application in a
  transport-independent way to the degree possible.
  This enables applications written to a single API
  to make use of transport protocols in terms of the features 
  they provide.

- A unified interface to datagram and connection-oriented transports, allowing
  use of a common API for connection-establishment and closing.

- Message-orientation, as opposed to stream-orientation, using
  application-assisted framing and deframing where the underlying transport
  does not provide these.

- Asynchronous Connection establishment, transmission, and reception. 
  This allows concurrent operations during establishment and event-driven
  application interactions with the transport layer, in line with developments
  in modern platforms and programming languages;
  
- Selection between alternate network paths, also allowing applications to be

>   notified additional information is available about the networks over which 

Reviewer: English messy: However, I intended ... informing the selection ... rather than the other possibility of informing the app.

Author: (will resolve outside this thread, for some reason having a comment here breaks multiline suggestions)

> - Selection between alternate network paths, also allowing applications to be
>   notified additional information is available about the networks over which 
>   a connection can operate (e.g. Provisioning Domain (PvD) information {{RFC7556}}).  

Author: ```suggestion
- Selection between alternate network paths, using additional information about the 
  networks over which a connection can operate (e.g. Provisioning Domain (PvD) 
  information {{RFC7556}}) where available.  
```

>   an connetcion can operate (e.g. Provisioning Domain (PvD) information {{RFC7556}}).  

Reviewer: /a connection/

Author: ```suggestion
  a connection can operate (e.g. Provisioning Domain (PvD) information {{RFC7556}}).  
```

### Transport Property Types {#property-types}

> Each Transport Property has a one ot the basic types described in {{notation}}.

Reviewer: /ot/to/

Author: ```suggestion
Each Transport Property has a one of the basic types described in {{notation}}.
```

the `congestionControl` property also yields the value `false`.

An implementation of this interface must provide sensible defaults for Selection

> Properties. The suggested default values for each property below represent a

Reviewer: could simply remove /suggested/  entirely?

Author: ```suggestion
Properties. The default values for each property below represent a
```
